structuralcodes/codes/mc2010/_concrete_shear.py

```diff
@@ -0,0 +1,1598 @@
+"""A collection of shear formulas for concrete"""
+import warnings
+from math import pi, tan, sin, cos
+
+
+def epsilon_x(
+    E_s: float,
+    As: float,
+    Med: float,
+    Ved: float,
+    Ned: float,
+    z: float,
+    delta_e: float,
+) -> float:
+    """The maximum allowed shear resistance
+
+    fib Model Code 2010, eq. (7.3-16)
+
+    Args:
+        E (float): The E-modulus to the material in MPa
+        As (Float): The cross-section area of reinforcement in mm^2
+        Med (Float): The moment working on the material in Nmm
+        Ved (float): The shear force working on the material in N
```

Comment:
Add in the docstring that `Med` and `Ved` should be taken as positive quantities, although you take the absolute value in the calculations.

Reply:
Can you elaborate more about what you mean by this? :)